pkg/common/utils/utils.go

```diff
@@ -223,3 +223,35 @@ func QueryVolumeDetailsUtil(ctx context.Context, m cnsvolume.Manager, volumeIds
 	}
 	return volumeDetailsMap, nil
 }
+
+// LogoutAllSessions will loggout all vCenter sessions and disconnect vCenter client
+func LogoutAllvCenterSessions(ctx context.Context) {
+	log := logger.GetLogger(ctx)
+	log.Info("logging out vCenter session")
+	virtualcentermanager := cnsvsphere.GetVirtualCenterManager(ctx)
+	vCenters := virtualcentermanager.GetAllVirtualCenters()
+	managerInstanceMap := cnsvolume.GetAllManagerInstances(ctx)
+	for _, vc := range vCenters {
+		if vc.Client == nil {
+			continue
+		}
+		log.Info("closing idle vCenter session")
+		vc.Client.CloseIdleConnections()
+		// logout vCenter session for list-view
+		mgr, ok := managerInstanceMap[vc.Config.Host]
+		if ok && mgr != nil {
+			err := mgr.LogoutListViewVCSession(ctx)
+			if err != nil {
+				continue
+			}
+		}
+		log.Infof("Disconnecting vCenter client for host %s", vc.Config.Host)
+		err := vc.Disconnect(ctx)
+		if err != nil {
+			log.Errorf("Error while disconnect vCenter client for host %s. Error: %+v", vc.Config.Host, err)
```

Comment:
`log.Errorf("error while disconnect vCenter client for host %s. Error: %+v", vc.Config.Host, err)`